src/Sylius/Bundle/ApiBundle/Command/Account/ResetPassword.php

```diff
@@ -13,12 +13,15 @@
 
 namespace Sylius\Bundle\ApiBundle\Command\Account;
 
+use Sylius\Bundle\ApiBundle\Attribute\TokenAware;
+
+#[TokenAware]
 class ResetPassword
 {
     public function __construct(
-        public string $token,
-        public ?string $newPassword = null,
-        public ?string $confirmNewPassword = null,
+        public readonly string $token,
+        public readonly string $newPassword,
+        public readonly string $confirmNewPassword,
```

Comment:
```suggestion
        #[\SensitiveParameter]
        public readonly string $newPassword,
        #[\SensitiveParameter]
        public readonly string $confirmNewPassword,
```
Same w/ the one in core